feat(mrc): add volume backup guides

FabienHenon opened the pull request:
ref: #16714

<!--
Hello 👋 Thank you for submitting a Pull Request.

Have any questions? Check out the contributing docs at https://github.com/ovh/manager/blob/master/CONTRIBUTING.md

Please read the following before submitting:

- Keep your PR as small as possible to make easy reviews.
- Commits are signed-off.
- Update only Messages_fr_FR.json locales for any content changes.
- Lint has passed locally.
- Standalone app was ran and tested locally.
- Ticket reference is mentioned in linked commits (internal only).
- Breaking change is mentioned in relevant commits.
- Limit your PR to one type (build, ci, docs, feat, fix, perf, refactor, revert, style, test or release)
-->

## Description

<!-- Write a brief description of the changes introduced by this PR -->
Add guides for volume backup

<!-- Provide Jira ticket or Github issue -->
Ticket Reference: #16714

## Additional Information

<!-- 
Mention any other information relevant to the PRs. It can be,

- link dependencies of PR
- Translations ticket reference
- Screenshots to better explain the change
 -->

sachinrameshn commented:
I will let control team review this before as I see linked manager issue is not yet reviewed from them.

anooparveti reviewed (changes requested):
**nitpick:** I dont like the idea of adding guides for PCI in MRC. Is it something that can be added on PCI COmmon?

FabienHenon commented:
> **nitpick:** I dont like the idea of adding guides for PCI in MRC. Is it something that can be added on PCI COmmon?

@anooparveti there are already a lot of PCI guides in this component (all of them I think). The entire component, maybe, should be moved to pci common. But it will break a lot of apps as soon as they update their MRC deps. Should I proceed?

anooparveti reviewed (changes requested):
We are not supposed to updated locale files apart from `fr_FR`.
Can you please get the changes from the translation team?

FabienHenon commented:
> We are not supposed to updated locale files apart from `fr_FR`. Can you please get the changes from the translation team?

@anooparveti these were existing translations from the manager. I just moved them inside MRC and changed the key to be alike the other keys.
Translations themselves have not changed and existed before
